lightkurve/tests/test_utils.py

```diff
@@ -106,3 +107,11 @@ def test_import():
     """Regression test for #605; `lk.utils` resolved to `lk.seismology.utils`"""
     from .. import utils
     assert hasattr(utils, "btjd_to_astropy_time")
+
+
+def test_btjd_bkjd_input():
+    """Regression test for #607: are the bkjd/btjd functions tolerant?"""
+    for user_input in [0, [0], np.array([0])]:
+        assert_array_equal(bkjd_to_astropy_time(user_input).value, np.array([2454833.]))
+    for user_input in [0, [0], np.array([0])]:
+        assert_array_equal(btjd_to_astropy_time(user_input).value, np.array([2457000.]))
```

Comment:
It might be a nitpick... But if the input is a scalar (case `btjd == 0` in the test), I think typical developers would expect the output to be a scalar as well.

Reply:
That's a fair comment.  Let's make it behave that way!